src/main/java/housing/Region.java

```diff
@@ -50,6 +61,21 @@ public Region(int targetPopulation) {
         regionalHouseholdStats = new RegionalHouseholdStats(this);
         regionalHousingMarketStats = new RegionalHousingMarketStats(houseSaleMarket);
         regionalRentalMarketStats = new RegionalRentalMarketStats(regionalHousingMarketStats, houseRentalMarket);
+        failedBidderCounter=new HashMap<Id<Household>,Integer>();
```

Comment:
This is redundant with the same line within the init method.

Reply:
Don't we also need to initialize the failedBidderCounter when we need other variables?
Anyway, this HashMap will be removed and the information will be stored in Household stored.

Reply:
Ok.